docs/user/rstudio/ide/guide/publish/publishing.qmd

```diff
@@ -51,12 +48,15 @@ You can publish R Markdown (`.Rmd`) and Quarto (`.qmd`) documents to Posit Conne
 
 ## Publishing Shiny apps
 
-You can publish Shiny apps to Posit Connect or shinyapps.io.
+You can publish Shiny apps to Posit Connect, Posit Cloud, or shinyapps.io.
```

Comment:
Note L43, you use posit.cloud instead of Posit Cloud, which we've used elsewhere

Also the Publish to RPubs or Connect may possibly be a little confusing since those are not the only 2 choices now? but also okay if you want to leave that bit as is until GA